system/settings/settings.xml

```diff
@@ -517,26 +517,24 @@
           </constraints>
           <control type="spinner" format="integer" delayed="true"/>
         </setting>
-        <setting id="subtitles.font" type="string" label="14089" help="36185">
+        <setting id="subtitles.fontname" type="string" label="14089" help="36185">
           <level>1</level>
-          <default>arial.ttf</default>
+          <!-- KodiDefault, is a placeholder font family name for the default Kodi font -->
+          <default>KodiDefault</default>
```

Comment:
I am unsure how seriously we want to take rebranding but probably it is better to avoid the Kodi naming. Maybe SystemDefault, AppDefault or SubtitleDefault would work?

Reply:
good point i had forgotten about rebranding...
i think i will do as on `subtitles.charset` with capitals DEFAULT